maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -153,7 +160,7 @@ public void execute() throws MojoExecutionException {
 			}
 
 			// Expand Jar into target/classes
-			expandJar(bndJar, classesDir);
+			expandJar(bndJar, classesDir, buildContext);
```

Comment:
Sure. I also thought about it but I was not sure if you maybe wanted it to be static